zebrad/tests/common/configs/v1.0.0-beta.14.toml

```diff
@@ -0,0 +1,66 @@
+# Default configuration for zebrad.
```

Comment:
Is there any difference between `v1.0.0-beta.13.toml` and `v1.0.0-beta.14.toml`?
We only need to test files if they are different.